src/XPlot.Plotly/Main.fs

```diff
@@ -224,10 +234,21 @@ type Chart =
     /// Combine charts together and display as a single page in default browser
     static member ShowAll(charts:seq<PlotlyChart>)=
         let html = charts |> Seq.map (fun c->c.GetInlineHtml()) |> Seq.reduce (+)
-        let pageHtml = Html.pageTemplate.Replace("[CHART]", html)
+        let plotlysrc charts =
+            match charts |> Seq.tryHead<PlotlyChart> with
+            |Some s -> s.PlotlySrc
+            |None -> Html.DefaultPlotlySrc
```

Comment:
```suggestion
            | Some s -> s.PlotlySrc
            | None -> Html.DefaultPlotlySrc
```

Reply:
Great, done. Thinking of next adding 'WithFill' or something that would call Plotly.newPlot with {responsive: true} and set the div style so Chart.Show will fill page. https://plotly.com/javascript/configuration-options/#making-a-responsive-chart
yes/no/meh?